src/bootstrap/config.rs

```diff
@@ -790,7 +790,7 @@ impl Config {
             set(&mut config.rust_rpath, rust.rpath);
             set(&mut config.jemalloc, rust.jemalloc);
             set(&mut config.test_compare_mode, rust.test_compare_mode);
-            set(&mut config.llvm_libunwind, rust.llvm_libunwind);
+            config.llvm_libunwind = rust.llvm_libunwind;
```

Comment:
Is there a reason for this to not be a set call still? I would not expect it to need changes.

Reply:
Since `config.llvm_libunwind` is now an option, the set has to be dropped as it just does the assignment in case of `Some`, unwrapping the value